lib/rubycritic/source_control_systems/git.rb

```diff
@@ -52,6 +52,18 @@ def stashes_count
       def travel_to_original_state
         `git stash pop`
       end
+
+      def path_revisions_counts
+        `git log --name-only --format='' | sort | uniq -c`
```

Comment:
I'd like to add that we should scope this to the paths passed to rubycritic.
For example, if I do `rubycritic ./app` we shouldn't care about the `git log` of files outside of this dir, right?
(or maybe we do because of possible file moves?)